language-support/ts/daml-react/defaultLedgerContext.ts

```diff
@@ -82,8 +109,12 @@ export function useFetch<T extends object, K, I extends string>(template: Templa
  *
  * @return The fetched contract.
  */
-export function useFetchByKey<T extends object, K, I extends string>(template: Template<T, K, I>, keyFactory: () => K, keyDeps: readonly unknown[]): FetchResult<T, K, I> {
-  return ledgerContext.useFetchByKey(template, keyFactory, keyDeps);
+export function useFetchByKey<T extends object, K, I extends string>(
+  template: Template<T, K, I>,
+  keyFactory: () => K,
+  keyDeps: readonly unknown[]
```

Comment:
Can `prettier` be configured to had dangling commas in parameter lists? I found them to be a nice way to reduce diff noise in Scala (I'm not even sure whether that's feasible in TS itself, just asking).

Reply:
I think this is possible but need to check.

Reply:
Create a _.prettierrc_ file as follows:

```
trailingComma: all
```

[Docs.](https://prettier.io/docs/en/options.html)